setup.py

```diff
@@ -15,8 +15,6 @@
 
 install_ccompiler()
 
-VERSION = "4.0.3"
```

Comment:
this wasn't used anywhere anymore

Reply:
And I think when we released 4.0.3 we talked about removing it!